bazel/external/quiche.patch

```diff
@@ -67,31 +67,14 @@
  #include "absl/types/span.h"
  #include "http2/adapter/data_source.h"
  #include "http2/adapter/http2_protocol.h"
+ #include "http2/adapter/http2_visitor_interface.h"
```

Comment:
We're planning to back-port these patches to QUICHE, right?

Reply:
This was added in QUICHE so I needed to adjust the patch lines here.

Reply:
*nod* but I mean all of the diffs in this file. For example, are we planning to back-port the ssize_t typdef, the NO_ERROR and the default returns?

Reply:
+1, why not simply ban use of `ssize_t` in QUICHE via a presubmit?

Reply:
IIRC, it's used by nghttp somewhere that oghttp has to overload (or some such).

Reply:
Yeah, nghttp2 uses ssize_t. Envoy actually has a similar workaround in envoy/common/platform.h.

Reply:
I'm working on back-porting the fixes in the patch file. Some of these patches will be unnecessary in the next QUICHE update.

Reply:
Awesome, that's great to hear!